### PR TITLE
Enhancement Issue #5 - include Linux version

### LINKED ISSUE #5: Include Linux version from /etc/issue as part of the platform
The operating system version seems like part of "the platform" to me. /etc/issue is pretty standard for Linux (the presumed "native os" for mycroft), so it would be well to make a best effort to include that in the platform readout.

### PR DESCRIPTION
**Pull request by jrwarwick**
Extract Linux version from /etc/issue if it exists, clean and speak.
Also threw in "system software" as version check phrase in vocab file.
Fixes #5

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -79,6 +80,12 @@ def check_platform_build(self, message):
             self.enclosure.activate_mouth_events()
         else:
             self.speak_dialog('platform.build.none')
+        # If this is a mainstream Linux, include basic version info
+        try:
+            opsys = re.sub(r'\\[a-z]{1}', ' ', open("/etc/issue").readline())
+            self.speak('On operating system: ' + opsys)
+        except Exception as e:
+            self.log.warning('/etc/issue read failed. ' + e.__class__.__name__ + ': ' + str(e))
```
Comment (MatthewScholefield) [inline __init__.py]: How about using `log.exception(` here which auto-adds the exception info?

Comment (jrwarwick) [reply __init__.py]: I did not know about log.exception.  I was just matching what I saw on line 60. I assume that should be replaced to match in coming supplementary commit.

Comment (MatthewScholefield) [reply __init__.py]: Thanks. One last thing, the `as e` parts are no longer necessary.

Comment (jrwarwick) [reply __init__.py]: Thanks for the feedback. I have removed the `as e` parts. Also threw in one last sanity filter line.

**Comment by MatthewScholefield** [comment]
Double sorry for the double-delay xD. Looks good!

**Pull request merged**